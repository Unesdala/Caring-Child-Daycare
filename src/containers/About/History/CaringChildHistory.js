import React, { Component } from 'react';
import commonUtils from '../../../components/lib/commonUtils';
import DefaultCaringChildHistory from './CaringChildHistoryContent';
import DefaultFooter from '../../../components/Footer/Footer';

export class CaringChildHistory extends Component {
  constructor(props) {
    super(props);
    this.commonUtils = commonUtils;
  }

  componentDidMount() { this.commonUtils.setTitleAndScroll(''); }

  render() {
    return (
      <div className="page-content">
        <main>
          <DefaultCaringChildHistory />
        </main>
        <DefaultFooter />
      </div>
    );
  }
}

export default CaringChildHistory;
