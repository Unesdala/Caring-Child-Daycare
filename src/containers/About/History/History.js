import React, { Component } from 'react';
import commonUtils from '../../../components/lib/commonUtils';
import DefaultCaringChildHistoryContent from './HistoryContent';
import DefaultFooter from '../../../components/Footer/Footer';

export class CaringChildHistory extends Component {
  constructor(props) {
    super(props);
    this.commonUtils = commonUtils;
  }

  componentDidMount() { this.commonUtils.setTitleAndScroll('Caring Child Daycare\'s History'); }

  render() {
    return (
      <div className="page-content">
        <DefaultCaringChildHistoryContent />
        <DefaultFooter />
      </div>
    );
  }
}

export default CaringChildHistory;
