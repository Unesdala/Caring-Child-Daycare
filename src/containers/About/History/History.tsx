import HistoryContent from './HistoryContent';

export const History = (): JSX.Element => {
  document.title = 'Caring Child Daycare | Caring Child Daycare&apos;s History';  
  return (
    <div className="Site-content">
      <main className="history-main">
        <HistoryContent />
      </main>
    </div>
  );
};

export default History;
