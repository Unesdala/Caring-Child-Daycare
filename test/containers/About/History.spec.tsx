/* eslint-disable @typescript-eslint/no-explicit-any */

import { shallow } from 'enzyme';
import { History } from 'src/containers/About/History/History';
import HistoryContent from 'src/containers/About/History/HistoryContent';

describe('History', () => {
  const wrapper = shallow(<History />);
  it('renders', () => {
    expect(wrapper.find(HistoryContent).exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find(HistoryContent).dive().find('main.history-main--content').exists()).toBe(true);
  });
  
});
