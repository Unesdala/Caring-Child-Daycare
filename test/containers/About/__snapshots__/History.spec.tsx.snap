// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`History renders 1`] = `ShallowWrapper {}`;
